### PR TITLE
dashboard: update to 0619

### PR DESCRIPTION
**Pull request by HunDunDM**
### What problem does this PR solve?

dashboard: update to 0619

### What is changed and how it works?

dashboard: update to 0619

### Check List

Tests

- Manual test (add detailed scripts or steps below)

Code changes

- Has configuration change

Related changes

- Need to cherry-pick to the release branch

### CODE REVIEW

File: conf/config.toml [hunk 1]
```diff
@@ -127,3 +127,6 @@ location-labels = []
 ## When enabled, request will be proxied to the instance running Dashboard
 ## internally instead of result in a 307 redirection.
 # internal-proxy = false
+
+## When not disabled, frontend will report some data for analysis
```
Comment (breeswish) [inline conf/config.toml:131]:
```suggestion
## When not disabled, usage data will be sent to PingCAP for improving user experience.
```

File: server/config/config.go [hunk 2]
```diff
@@ -1181,6 +1181,7 @@ type DashboardConfig struct {
 	TiDBKeyPath      string `toml:"tidb-key-path" json:"tidb_key_path"`
 	PublicPathPrefix string `toml:"public-path-prefix" json:"public_path_prefix"`
 	InternalProxy    bool   `toml:"internal-proxy" json:"internal_proxy"`
+	DisableTelemetry bool   `toml:"disable-telemetry" json:"disable_telemetry"`
```
Comment (lhy1024) [inline server/config/config.go:1184]: Does the config option need more comments?

Comment (HunDunDM) [reply server/config/config.go:1186]: There are instructions in the code of `tidb-dashboard` and `config.toml`.

**Pull request merged**